azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/constant.py

```diff
@@ -0,0 +1,2 @@
+C2D = "c2d"
+INPUT = "input"
```

Comment:
is this really better than an enum? long term?

Reply:
Yeah, enums are relatively new to Python. This is standard practice to use string literals. The practice of a separate constants.py is less common, but makes sense given that it's specific to the package rather than a module.

---
In reply to: [259537543](https://github.com/Azure/azure-iot-sdk-python-preview/pull/49#discussion_r259537543) [](ancestors = 259537543)